src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -254,6 +385,11 @@ private sealed class RequestCompletionSource : TaskCompletionSource<HttpResponse
             public HttpRequestMessage RequestMessage { get; set; }
 
             public CurlHandler Handler { get; set; }
+
+            public CurlProxyUsePolicy ProxyPolicy { get; set; }
+
+            public IWebProxy Proxy { get; set; }
```

Comment:
Will fix this. @vijaykota  also pointed out the same thing. I was following the style of winhhtphandler which was doing similar. I will fix this in my next iteration (along with other comments).